astropy/nddata/nduncertainty.py

```diff
@@ -144,6 +144,13 @@ def unit(self):
                 return self.parent_nddata.unit
         return self._unit
 
+    @property
+    def quantity(self):
+        """
+        This uncertainty as an `~astropy.units.Quantity` object.
+        """
+        return Quantity(self.array, self.unit, copy=False)
```

Comment:
Note that in `Column`, we do
```
Quantity(self, copy=False, dtype=self.dtype, order='A')
```
not sure if this is needed here.

Reply:
I think the most natural thing in this case is to preserve *whatever* the input array is if at all possible, so ``order='K'`` makes sense.  Not sure if the ``dtype`` matters in this case since the input is an array... but better safe than sorry just to be sure.

Reply:
Yes, do add `dtype=self.dtype` - by default, things get converted to `float`.